asr/models/embedding.py

```diff
@@ -15,42 +94,58 @@ def load_embedding(fp, verbose=1):
         File path of the trained embedding vectors.
     verbose: int
         The verbosity. Default 1.
-
+    word_index: dict
+        Sample word embeddings.
+    
     Returns
     -------
     dict:
         The embedding weights stored in a dict with the word as key and
         the weights as values.
     """
-
-    embedding = {}
-
+    
+    #Granularity of the worker jobs.
+    block_size=1000
+    #Maximum number of jobs in the queue.
+    queue_size=500
+    #Number of worker jobs running in parallel.
+    n_worker=3
```

Comment:
Make this an argument named `n_jobs` (sklearn). Default 1. Value -1 should be all cores.